_docs/Contrib.md

```diff
@@ -1,5 +1,86 @@
-* All components should implement the related native HTML component attributes if one exists
-* All input components which are an implementation of an HTML component should have the native input element rendered by them (for SEO and a11y)
+## Accessibility
+
+### Introduction
+
+Implementing accessibility for a component will be done according to the following checklist:
+1) **WAI-ARIA role**: a role markup attribute (i.e. role="button") which sets the type of the component. Attaching a role gives assistive technologies information about how to handle the element. Some roles can be achieved automatically by using HTML5 section elements (such as nav, form). For a full listing of roles see [Roles Categorization](https://www.w3.org/TR/wai-aria-1.1/#roles_categorization).
+2) **WAI-ARIA state & property**: aria-prefixed markup attribute which sets the state of a component (i.e. "aria-checked" for a checkbox) or a property of it (i.e. aria-label). When combined with a role, it helps the assistive technology convey information to the user. Note that the difference between a state and property is semantic when considering the code implementation - states usually change a lot while properties do not. For a full listing see [States & Properties](https://www.w3.org/TR/wai-aria-1.1/#states_and_properties).
```

Comment:
unclear if it means using WAI-ARIA on the children or exposing it in props
not  explicitly mentioned in the code example below